<!-- This is an example launch file for how a ROS application could use a node -->
<launch>
   
	<arg name="instance_id" default="default"/>
	<param name="instance_id" value="$(arg instance_id)"/>
    <param name="test_stt" value="false"/>
    <arg name="test_input" default="$(find harmoni_stt)/test_data/hello.wav"/>
    <param name="test_input_stt" value="$(arg test_input)"/>
    <rosparam file="$(find harmoni_stt)/config/google_configuration.yaml" subst_value="True"/>
    <node pkg="harmoni_stt" type="google_service.py" name="harmoni_stt_$(arg instance_id)" output="screen"/>

</launch>